### PR TITLE
feat: add playwright config documentation

### PR DESCRIPTION
**Pull request by ferrandiaz**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [x] Code is linted (`npm run lint`)

## Notes for the Reviewer
Documentation for supported options of playwright config

### CODE REVIEW

**Review by tnolet — APPROVED** [review]
@ferrandiaz I made some tweaks. Here are some tips for the future. Nice work on the matrix

1.  Capitalize brand names. Checkly and Playwright.
2. Reduce duplication.

File: site/content/docs/browser-checks/playwright-test.md [hunk 1]
```diff
@@ -109,11 +107,197 @@ Here's an example of a Playwright Test script that fails, and provides a video o
...
+## Global configuration
+
+We are gradually rolling out support for global configuration options for the Playwright Test Runner. This allows you to 
+configure your Playwright tests in a single place, instead of having to repeat the same configuration for each test file.
+
+{{< info >}}
+This feature is in **beta** and is only available when using the Checkly CLI to manage your checks and associated `.spec.ts|js` files.
+{{< /info >}}
+
+There three things you should be aware of:
```
Comment (ferrandiaz) [inline site/content/docs/browser-checks/playwright-test.md:128]:
```suggestion
There are three things you should be aware of:
```

File: site/content/docs/browser-checks/playwright-test.md [hunk 2]
```diff
@@ -109,11 +107,197 @@ Here's an example of a Playwright Test script that fails, and provides a video o
...
+
+
+If you have an existing `playwright.config.ts` or `playwright.config.js` file in your project that you want to import, 
+you can simply run the [sync-playwright](/docs/cli/command-line-reference/#npx-checkly-sync-playwright)
+
+```bash
+npx checkly sync-playwright
+```
+
+This command will all currently supported Playwright config option to your `checkly.config.ts` file.
```
Comment (ferrandiaz) [inline site/content/docs/browser-checks/playwright-test.md:141]:
```suggestion
This command will add currently supported Playwright config option to your `checkly.config.ts` file.
```

**Review by tnolet — APPROVED** [review]
LGTM

File: site/content/docs/cli/command-line-reference.md [hunk 3]
```diff
@@ -220,3 +220,12 @@ checkly env rm [KEY] [-f]
 
 - `KEY`: Environment variable key.
 - `--force` or `-f`: Skips the confirmation dialog when removing an environment.
+
+
+### `npx checkly sync-playwright`
```
Comment (tnolet) [inline site/content/docs/cli/command-line-reference.md:225]:
```suggestion
## `npx checkly sync-playwright`
```

File: site/content/docs/cli/command-line-reference.md [hunk 4]
```diff
@@ -220,3 +220,12 @@ checkly env rm [KEY] [-f]
...
+
+Copy Playwright config into the Checkly config file
```
Comment (tnolet) [inline site/content/docs/cli/command-line-reference.md:227]:
```suggestion
Copy Playwright config into the Checkly config file.
```

**Review by tnolet — CHANGES_REQUESTED** [review]
pay attention to the document structure and end sentences with a .

**Pull request merged**